helix-term/src/commands/lsp.rs

```diff
@@ -1080,11 +1080,17 @@ pub fn goto_implementation(cx: &mut Context) {
 pub fn goto_reference(cx: &mut Context) {
     let (view, doc) = current!(cx.editor);
     let language_server = language_server!(cx.editor, doc);
+    let config = cx.editor.config.load();
```

Comment:
```suggestion
    let config = cx.editor.config();
```

There will be a borrowing conflict for `cx.editor`. Moving this above the `let (view, doc)` line will resolve that

Reply:
Sorry, I am just learning Rust. Can you please elaborate about the borrowing conflict? It works fine right now.

If I remove the call of load method and put it above then it will not be compiled.